testnets/_IBC/junotestnet-osmosistestnet.json

```diff
@@ -28,6 +28,38 @@
         "dex": "osmosis"
       }
     }
-  ]
-}
-  
\ No newline at end of file
+  ],
+    {
+      "chain_1": {
+        "chain_name": "junotestnet",
+        "client_id": "07-tendermint-539",
+        "connection_id": "connection-671"
+      },
+      "chain_2": {
+        "chain_name": "osmosistestnet",
+        "client_id": "07-tendermint-832",
+        "connection_id": "connection-739"
+      },
+      "channels": [
+        {
+          "chain_1": {
+            "channel_id": "channel-622",
+            "port_id": "transfer"
```

Comment:
I'm surprised to see the transfer port id, when I'm used to seeing that cw20s need a contract and wasm.contract port for moving cw20s

Reply:
I think juno factory token is not follow cw20 format.

Reply:
That's correct, my mistake